apps/web/vibes/2px/components/dropdown/index.tsx

```diff
@@ -0,0 +1,94 @@
+'use client'
+
+import { useState } from 'react'
+
+import clsx from 'clsx'
+
+import { ChevronDownIcon } from '../icons'
+
+interface OptionProps {
+  label: string
+  value: string
+}
+
+interface DropdownProps {
+  placeholder: string
+  options: OptionProps[]
+  status?: 'normal' | 'error' | 'success'
+  disabled?: boolean
+  value?: string
+  setValue: (value: string) => void
+  className?: string
+}
+
+const Dropdown = ({
+  placeholder,
+  options,
+  status = 'normal',
+  disabled,
+  setValue,
+  value,
+  className,
+}: DropdownProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className={clsx('relative w-full font-body text-foreground @container', className)}>
+      <button
+        className={clsx(
+          'relative z-0 flex w-full items-center justify-between border-2 px-4 py-[1.125rem] text-base font-medium leading-6 outline-none @lg:px-5 @lg:py-[1.375rem] @lg:text-xl @lg:leading-9',
+          {
+            'border-error': status === 'error',
+            'border-success': status === 'success',
+            'border-foreground': status === 'normal',
+            'cursor-not-allowed border-contrast-300 text-contrast-300': disabled,
+            'hover:border-dashed focus:border-dashed': !disabled,
+            'border-dashed': isOpen,
+          }
+        )}
+        disabled={disabled}
+        onClick={() => setIsOpen(!isOpen)}
+        type="button"
+        aria-label={placeholder}
+      >
+        {options.find(option => option.value === value)?.label || placeholder}
+
+        <ChevronDownIcon className="stroke-foreground" />
```

Comment:
The chevron icon should decrease in size for smaller screens.

Reply:
Actually it's the same size on both small and large screens

Reply:
@LHDi In [the design file](https://www.figma.com/design/JkWOwdQdlnrc04tUy3zlMK/Vibes---2px?node-id=5341-4817&m=dev) it seems like the icon is a 1rem x 1rem square for smaller screens while 1.5rem x 1.5rem for bigger screens.

Am I missing something?

Reply:
You are right! and I am right! lol

I've been following the icon sizes of the open version.
![image](https://github.com/user-attachments/assets/a620fb7b-7004-4672-95bc-cbf48aad8d69)

I'll follow yours, it makes more sense 🫡